frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -28,4 +28,23 @@ def update_genders_and_salutations():
 				# make sure DuplicateEntryError is for the exact same doc and not a related doc
 				pass
 			else:
-				raise
\ No newline at end of file
+				raise
+
+def update_global_search_doctypes():
+	global_searches_doctypes = ["Web Page", "Webhook", "Google Contacts", "Energy Point Rule", "Address", "Contact", "Event",
```

Comment:
Remove Single types and keep DocTypes which makes sense to be searched like Address, Contact, Web Page, etc.